lib/ramble/ramble/repository.py

```diff
@@ -1052,7 +1055,7 @@ def __contains__(self, app_name):
         return self.exists(app_name)
 
 
-def create_repo(root, namespace=None):
+def create_repo(root, namespace=None, subdir=applications_dir_name):
```

Comment:
Should we make `applications_dir_name` more obviously `const` (ie enum member or similar) if we're making such that a user changing the dir does not update the var?